profiling/default.nix

```diff
@@ -0,0 +1,21 @@
+{ pkgs,config }:
+let
+ name = "hc-sim2h-server-perf";
+  script-sim2h-server-perf = pkgs.writeShellScriptBin name
+  ''
+  cd crates/sim2h_server && cargo build -p sim2h_server --release && perf record --call-graph dwarf sim2h_server
```

Comment:
this command needs to pass arbitrary parameters through to sim2h_server because I almost never start it just plain.

Reply:
Will change this